apierror.h

```diff
@@ -68,6 +70,6 @@
 /*! \brief Helper method to get a string representation of an API error code
  * @param[in] error The API error code
  * @returns A string representation of the error code */
-const char *janus_get_api_error(int error);
+shared const char *janus_get_api_error(int error);
```

Comment:
Hi Lorenzo, "shared" is just #define defined in os.h
#ifdef _WIN32
#ifdef SHARED
#define shared __declspec(dllexport) 
#else
#define shared __declspec(dllimport)
#endif
#else
#define shared
#endif
It is empty on other OSes (no affect). It is used to mark particular object whether it should be imported to dll or exported.